src/main/java/io/fabric8/elasticsearch/plugin/kibana/KibanaSeed.java

```diff
@@ -105,12 +98,30 @@ public static void setDashboards(String user, Set<String> projects, Set<String>
 		else {
 			List<String> common = new ArrayList<String>(indexPatterns);
 			
+			// build normalizedSort
+			List<String> normalizedSort = new ArrayList<String>();
```

Comment:
Diamond operator ftw! Generic types inferred from declaration. Look out for more inference to come in Java in the future hopefully, e.g. proposed http://openjdk.java.net/jeps/286